superset/examples/configs/datasets/examples/FCC_2018_Survey.yaml

```diff
@@ -1320,7 +1320,7 @@ columns:
   expression: null
   description: null
   python_date_format: null
-- column_name: curr_emplymnt_other
+- column_name: curr_employment_other
```

Comment:
I think we need to double check that this doesn't break examples creation, as I assume this is the actual name of the column in the dataset..

Reply:
good catch, yes it is the column name